test/byContract/IexecERC20/IexecERC20.ts

```diff
@@ -0,0 +1,271 @@
+// SPDX-FileCopyrightText: 2020-2024 IEXEC BLOCKCHAIN TECH <contact@iex.ec>
+// SPDX-License-Identifier: Apache-2.0
+
+import { loadFixture } from '@nomicfoundation/hardhat-network-helpers';
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/signers';
+import { BigNumber } from 'ethers';
+import { ethers, expect } from 'hardhat';
+import { loadHardhatFixtureDeployment } from '../../../scripts/hardhat-fixture-deployer';
+import {
+    IexecInterfaceNative,
+    IexecInterfaceNative__factory,
+    TestReceiver,
+    TestReceiver__factory,
+} from '../../../typechain';
+import { NULL } from '../../../utils/constants';
+import { getIexecAccounts } from '../../../utils/poco-tools';
+import { IexecWrapper } from '../../utils/IexecWrapper';
+
+const emptyValue = 0;
+const value = 100;
+const zeroAddress = NULL.ADDRESS;
```

Comment:
Use `ethers.constants.AddressZero` instead.

Reply:
Lol I used that but changed it for the review!